x-pack/plugins/monitoring/server/lib/cluster/get_index_patterns.ts

```diff
@@ -41,6 +42,61 @@ export function getIndexPatterns(
       };
     }, {}),
   };
-
   return indexPatterns;
 }
+
```

Comment:
new function similar to `getIndexPatterns` but returns 1.  There will be a case for each type (kibana, elasticsearch, logstash, etc).  Since the datastreams use `elasticsearch` instead of `es` (used in `.monitoring-es`) we have to map them.